src/main/java/frc/team2412/robot/commands/diagnostic/diagnosticSequentialCommand.java

```diff
@@ -0,0 +1,22 @@
+package frc.team2412.robot.commands.diagnostic;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.team2412.robot.subsystems.IntakeSubsystem;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+
+public class diagnosticSequentialCommand extends SequentialCommandGroup {
+	private final LauncherSubsystem launcherSubsystem;
+	private final IntakeSubsystem intakeSubsystem;
+
+	public diagnosticSequentialCommand(
+			IntakeSubsystem intakeSubsystem, LauncherSubsystem launcherSubsystem) {
```

Comment:
Another white space inconsistency.

Reply:
![image](https://github.com/robototes/Crescendo2024/assets/145405364/f8ddbd7e-8c83-43e0-8301-7dc456a5e212)
heres how it looks on my end, is it a github issue?

Reply:
Looks to be the way spotlesss wants it so will live with it.